README.md

```diff
@@ -270,6 +270,70 @@ abigen!(
 );
 ```
 
+### Multi-contract calls
+
+Sometimes, you might need to call your contract, which calls other contracts. To do so, you must feed the external contract IDs that your contract depends on to the method you're calling. You do it by chaining `.set_contracts(&[external_contract_id, ...])` to the method you want to call. For instance:
+
+```Rust
+let response = my_contract
+    .my_method(...)
+    .set_contracts(&[another_contract_id]) // Add this to set the external contract
+    .call()
+    .await
+    .unwrap();
+```
+
+Here's a more concrete example:
+
+```Rust
```

Comment:
This is why moving all these docs into The Fuels-rs Book is a `good idea`: you could have them as real files then just include them.